docs/graphs/graph-sdk-example.md

```diff
@@ -0,0 +1,289 @@
+---
+title: Graph SDK Example
+sidebar_position: 100
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+
+## Example Overview
+
+This example demonstrates the ability of graph databases in analyzing and offering recommendations for items that are commonly purchased together. By establishing connections between customer and item vertices using edge collections, the graph is able to seamlessly and accurately deliver tailored recommendations to customers who have bought a particular item.
+
+![Create a collection](/img/graphs/sdk-graph-image.png)
+
+In the graph that we created customers Dino and Durga bought the same items. The graph is able to recommend items to Dallas based on the items that Dino and Durga bought. This is a simple example of how graph databases can be used to offer recommendations to customers.
+
+## How to define vertices and edges
+
+For this example we will have two vertices - `groceryCustomers` and `groceryItems` and one edge collection - `groceryOrders`. The `groceryOrders` edge collection will be used to establish a connection between a customer and an item that they bought. Vertex collections are document store collections that store the vertices of a graph. Edge collections are graph edge collections that store the edges of a graph, edge collections have the `_from` and `_to` attributes that are used to establish a connection between two vertices.
+
+Vertex example in `groceryCustomers` collection:
+
+```json
+{
+    "_key": "C01",
+    "name": "John Doe"
+},
+...
+```
+
+Vertex example in `groceryItems` collection:
+
+```json
+{
+    "_key": "P03",
+    "name": "pineapple"
+},
+...
+```
+
+Graph edge example in `groceryOrders` collection (Connection between customer C01 and item P03):
+
+```json
+{
+    "_from": "groceryCustomers/C01",
+    "_key": "4117657795",
+    "_to": "groceryItems/P03"
+},
+...
+```
+
+## Import the example data
+
+If you want to try out the example, you can import the data into your Macrometa GDN account.
+Data can be found in the `sample-data` folder of this [repository](https://github.com/Macrometacorp/macrometa-graph-example).
+
+- Create two document collections in Macrometa GDN using console.
+    - `groceryItems`
+    - `groceryCustomers`
+
+- Create one edge collection in Macrometa GDN using Macrometa console.
+    - `groceryOrders`
+
+- Import test data from sample-data folder to created collections using console. Name of the files are the same as the collection names.
```

Comment:
Link to instructions for this. Also, do they need to select a primary key when they import?